packages/DataHeader/src/DataHeader.styles.js

```diff
@@ -36,15 +36,11 @@ export const Header = styled.div(({ $color, $backgroundColor, refHeader }) => {
     ${$backgroundColor || $color ? resetHeaderParentContainer() : ""}
 
     [data-pka-anchor="raw-button"] {
-      padding: 0 2px;
+      border-radius: ${tokens.button.borderRadius};
+      padding: 1px 2px;
```

Comment:
😅 thta's some awkward padding

Reply:
to make the focus ring(inset shadow) not too close to the SVG icon, if it's bigger, the shadow will be cut off by parent's padding